src/components/Editor/index.js

```diff
@@ -276,30 +277,50 @@ const Editor = React.createClass({
     this.previewSelectedToken(e, modifiers);
   },
 
+  async getSelectedExpression(token) {
```

Comment:
this looks like a great util that could move to `utils/editor`... and then be tested:

```js
async getSelectedExpression(token, cm, {
  selectedFrame, pauseData, sourceText, selectedToken
}) {
...
}
```

we could also move `getExpressionFromToken` as well...